virtualizarr/tests/test_backend.py

```diff
@@ -502,3 +506,66 @@ def test_open_dataset_with_scalar(self, hdf5_scalar, hdf_backend):
         with open_virtual_dataset(hdf5_scalar, backend=hdf_backend) as vds:
             assert vds.scalar.dims == ()
             assert vds.scalar.attrs == {"scalar": "true"}
+
+
+@requires_hdf5plugin
+@requires_imagecodecs
+@parametrize_over_hdf_backends
+@pytest.mark.parametrize(
+    "parallel",
+    [
+        False,
+        ThreadPoolExecutor,
+        pytest.param(DaskDelayedExecutor, marks=requires_dask),
+        pytest.param(
+            LithopsEagerFunctionExecutor,
+            marks=[
+                requires_lithops,
+                pytest.mark.xfail(
+                    reason="Lithops bug - see https://github.com/lithops-cloud/lithops/issues/1428"
```

Comment:
See https://github.com/lithops-cloud/lithops/issues/1428

Reply:
I was able to dodge this by not using `functools.partial`, but I'm a bit worried that my newer approach of using a closure won't work properly in a remote execution context.